docs/documentation/pages/internal/DEVELOPMENT.md

```diff
@@ -380,6 +380,35 @@ Usage:
 {{- end }}
 ```
 
+### Container images
```

Comment:
Does not seem related to this PR.

Reply:
This is because we now use helm image helpers in all controllers, so we need a devel doc for it.